packages/oui-button/_mixins.less

```diff
@@ -355,6 +355,12 @@
     }
   }
 
+  .button-icon-external-link(
```

Comment:
A mixin just for one property ?

Reply:
I did it to avoid setting this 0 margin into button.less file, since there is no such functions in, but it isn't necessary ?

Reply:
We avoid creating mixins just for one use, called only once or for this kind of use. 
You can directly put this in the selector.